subscription-sender/internal/queue/consumer.go

```diff
@@ -14,23 +13,23 @@ type Broker interface {
 	Drain() error 
 }
 
-type NotificationService interface {
-	NotifySubscribers(ctx context.Context, notification services.Notification )
+type notificationService interface {
+	HandleMessage(msg []byte, ctx context.Context)
 }
 
 type SubscriptionNotifierConsumer struct {
 	Broker Broker
-	subscriptionService NotificationService
+	messagesService notificationService
```

Comment:
Why does the naming is differs?
Seems like not clean logic if your name and interface do not the same what they meen

Reply:
done